core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -257,24 +256,31 @@ trait WhiskActionsApi extends WhiskCollectionAPI with PostActionActivation with
     onComplete(invokeAction(user, actionWithMergedParams, payload, waitForResponse, cause = None)) {
       case Success(Left(activationId)) =>
         // non-blocking invoke or blocking invoke which got queued instead
-        complete(Accepted, activationId.toJsObject)
+
+        respondWithActivationIdHeader(activationId) {
+          // note that if header defined a content-type, it will be ignored
+          // since the type must be compatible with the data response
```

Comment:
I don’t think this comment makes sense here.

Reply:
removed.